doc/Frontend/ideas.org

```diff
@@ -78,7 +75,81 @@
   + cons
     * A few extra characters for every signature
     * How does sugar get added from this system? (Jeremy)
+* Module Syntax
+Currently there are two roads we can go down in regards to module
+syntax.
+
+- mariari :: I personally prefer the "like functions" syntax as it
```

Comment:
I generally agree, I actually wonder if there is a strong reason to distinguish "modules" and "records" at all - is there?

Reply:
The only reason is for syntactic beauty.

A record would have { } before it and ,/;'s to delineate separation.

Also records, have to be pre-declared, whereas a module is a bit more free form

Reply:
Aye, makes sense.

Reply:
Maybe we should make it more different, if it can't be unified with records.